src/test/reporters/json.ts

```diff
@@ -65,11 +66,18 @@ export interface JSONReportTestResult {
   status: TestStatus | undefined;
   duration: number;
   error: TestError | undefined;
-  stdout: JSONReportSTDIOEntry[],
-  stderr: JSONReportSTDIOEntry[],
+  stdout: JSONReportSTDIOEntry[];
+  stderr: JSONReportSTDIOEntry[];
   retry: number;
   steps?: JSONReportTestStep[];
-  attachments: { name: string, path?: string, body?: string, contentType: string }[];
+  attachments: {
+    name: string;
+    path?: string;
+    body?: string;
+    contentType: string;
+  }[];
+  line?: number;
```

Comment:
Let's instead do `errorLocation?: { line: number, column: number }`.

Reply:
I feel strongly that test result does not have a line/column because it is not defined in the source like suite or test. `errorLocation` works nicely here, in line with https://playwright.dev/docs/api/class-location.

Reply:
Done.

Just curious, shouldn't `JSONReportSuite` and `JSONReportSpec` be using the Location class as `file`, `line` and `column` are being used there?